packages/KPrompt/KPrompt.vue

```diff
@@ -66,6 +67,12 @@
             :disabled="disableProceedButton"
             class="k-prompt-proceed"
             @click="proceed">
+            <KIcon
+              v-if="actionPending"
+              slot="icon"
+              icon="spinner"
+              size="16"
+            />
```

Comment:
nit: The icon looks a little small in the button and the gray color is a little hard to see when the button is disabled

![image](https://user-images.githubusercontent.com/2229946/155199084-6472dc67-61a4-470e-aa03-9fc45f73fafd.png)

Reply:
The font size is 16, I can't make the spinner any bigger without getting weird positioning of things. I DID change the color to white to improve visibality.

![image](https://user-images.githubusercontent.com/67973710/155202162-dc9b47de-184a-494b-afd8-e8e237479c5b.png)